cache.js

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import LRU from 'lru-cache';
+
+class Cache {
```

Comment:
I'm noticing that this class is barely useful ... it just makes a couple of *very* minor changes to the LRU API. Why not just use the LRU API instead?

Reply:
Removed.